aomovie.cpp

```diff
@@ -28,16 +26,16 @@ void AOMovie::play(QString p_gif, QString p_char, QString p_custom_theme)
 
   QString custom_path;
   if (p_gif == "custom")
-    custom_path = ao_app->get_image_suffix(ao_app->get_character_path(p_char, p_gif));
+    custom_path = TextFileHandler::getInstance().get_image_suffix(TextFileHandler::getInstance().get_character_path(p_char, p_gif));
   else
-    custom_path = ao_app->get_image_suffix(ao_app->get_character_path(p_char, p_gif + "_bubble"));
-
-  QString misc_path = ao_app->get_base_path() + "misc/" + p_custom_theme + "/" + p_gif + "_bubble.gif";
-  QString custom_theme_path = ao_app->get_custom_theme_path(p_custom_theme, p_gif + ".gif");
-  QString theme_path = ao_app->get_theme_path(p_gif + ".gif");
-  QString default_theme_path = ao_app->get_default_theme_path(p_gif + ".gif");
-  QString placeholder_path = ao_app->get_theme_path("placeholder.gif");
-  QString default_placeholder_path = ao_app->get_default_theme_path("placeholder.gif");
+    custom_path = TextFileHandler::getInstance().get_image_suffix(TextFileHandler::getInstance().get_character_path(p_char, p_gif + "_bubble"));
+
+  QString misc_path = TextFileHandler::getInstance().get_base_path() + "misc/" + p_custom_theme + "/" + p_gif + "_bubble.gif";
+  QString custom_theme_path = TextFileHandler::getInstance().get_custom_theme_path(p_custom_theme, p_gif + ".gif");
+  QString theme_path = TextFileHandler::getInstance().get_theme_path(p_gif + ".gif");
+  QString default_theme_path = TextFileHandler::getInstance().get_default_theme_path(p_gif + ".gif");
+  QString placeholder_path = TextFileHandler::getInstance().get_theme_path("placeholder.gif");
+  QString default_placeholder_path = TextFileHandler::getInstance().get_default_theme_path("placeholder.gif");
```

Comment:
You might want to put it into a variable...

Reply:
I'll probably do that then in some other places where this kinda thing happens.